fio-plot/bench_fio

```diff
@@ -0,0 +1,494 @@
+#!/usr/bin/env python3
```

Comment:
If the fio-plot code is copy-pasted to this repo, the LICENSE file(s) need to be included as well. Alternatively, can the upstream fio-plot repo be cloned in the Dockerfile?

Reply:
The upstream repo contains output images committed so the size is very large, and that would make our image size > 200 MB. So, I preferred just adding useful files and making an image out of it. The upstream repo also has lot of unnecessary files.

Reply:
can those be removed? the main drawback of copy-pasting code is that updating is a manual task, whereas cloning allows for easy updating. We could do:

```
ARG VERSION
RUN git clone --branch=${VERSION} <fio-plot>
         pip install reqs
         rm -r /paths/that /we/dont  /want/to/persist
```

Reply:
I see. Yeah, that can be done. ✅  updating it soon. Thanks.